packages/shorebird_cli/lib/src/validators/shorebird_version_validator.dart

```diff
@@ -11,10 +11,11 @@ class ShorebirdVersionValidator extends Validator {
   final Future<bool> Function({required String workingDirectory})
       isShorebirdVersionCurrent;
 
-  // coverage:ignore-start
   @override
   String get description => 'Shorebird is up-to-date';
-  // coverage:ignore-end
+
+  @override
+  ValidatorScope get scope => ValidatorScope.installation;
```

Comment:
can a validator have multiple scopes?

Reply:
I don't think so? If a validator is checking both project setup and global installation setup, it should probably be split into two validators.